param_templates/input_nml.yaml

```diff
@@ -58,6 +58,10 @@ mpp_io_nml:
         values: .true.
 
 nam_stochy:
+    stochini:
+        values:
+          $CONTINUE_RUN == True: .true.
```

Comment:
@mnlevy1981 are you suggesting this addition?:
$RUN_TYPE == "branch": .true.

Reply:
I think so! As it stands, it looks to me like `stochini` won't be set in a branch run, but it should be.

Reply:
I'll try an ERI test and let you know.